src/cljc/athens/common_events.cljc

```diff
@@ -246,6 +251,32 @@
                      :new-uid new-uid}}))
 
 
+(defn build-left-sidebar-drop-above
```

Comment:
Please try to keep similar events together, these belong next to other shortcut events.